docs/guide/architecture-styles/event-driven-content.md

```diff
@@ -30,6 +30,12 @@ There are two primary topologies within many event-driven architectures:
 
 - **Mediator topology**. This topology addresses some of the shortcomings of broker topology. There is an event mediator that manages and controls the flow of events. The event mediator maintains the state and manages error handling and restart capabilities. Unlike broker topology, components broadcast occurrences as commands and only to designated channels, usually message queues. These commands aren't expected to be ignored by their consumers. This topology offers more control, better distributed error handling, and potentially better data consistency. This topology does introduce increased coupling between components, and the event mediator could become a bottleneck or a reliability concern.
 
+Sometimes, the event producer requires an immediate response from the event consumer, such as obtaining a customer ID before proceeding with an order. In event-driven architecture, synchronous communication can be implemented in two main ways:
+
+- **Create two distinct queues.** a request queue and a reply queue. The event producer sends an asynchronous request to the request queue, pauses other operations, and waits for a response from the reply queue. Event consumers then process the request and send the reply back through the reply queue. This technique is more common and utilizes a correlation ID for tracking.
+  
+- **Create a dedicated ephemeral queue for each request.** This approach also pauses other operations by the event producer. Once the request is complete, the queue is deleted. While this technique is simpler, it lacks scalability, as the message broker must create and delete a queue for each request. 
```

Comment:
I don't understand this one.  Let's keep it simple, and use Azure Storage queues as the example.  How would consumers know about this new queue to start monitoring?  Who deletes the queue (the consumer?  The publisher?).  I think this is too vague, and you're going to need to say a bit more.